app/styles/layout.scss

```diff
@@ -0,0 +1,34 @@
+.container {
+  max-width: 60rem;
+  margin: 0 auto;
+}
+
+.grid {
+  display: flex;
```

Comment:
Add margin -1rem to pull the grid items back in line with things that aren't part of the grid.